common/install_cuda.sh

```diff
@@ -67,10 +67,10 @@ function install_121 {
 
     # NCCL license: https://docs.nvidia.com/deeplearning/nccl/#licenses
     mkdir tmp_nccl && cd tmp_nccl
-    wget -q https://developer.download.nvidia.com/compute/redist/nccl/v2.18.1/nccl_2.18.1-1+cuda12.1_x86_64.txz
-    tar xf nccl_2.18.1-1+cuda12.1_x86_64.txz
-    cp -a nccl_2.18.1-1+cuda12.1_x86_64/include/* /usr/local/cuda/include/
-    cp -a nccl_2.18.1-1+cuda12.1_x86_64/lib/* /usr/local/cuda/lib64/
+    wget -q https://developer.nvidia.com/downloads/compute/machine-learning/nccl/secure/2.19.3/agnostic/x64/nccl_2.19.3-1+cuda12.0_x86_64.txz
```

Comment:
```suggestion
    wget -q https://ossci-linux.s3.amazonaws.com/nccl_2.19.3-1%2Bcuda12.0_x86_64.txz
```

Reply:
The secure URL may not be available. Hence lets use same version uploaded to pytorch s3 path.